### PR TITLE
Restoring-window view and automatic save points russian translation

### PR DESCRIPTION
**Pull request by Jollfye**
Made "js_history_window" and "js_history_tab" to start with capital letter according with latest English changes [here](https://github.com/deanoemcke/thegreatsuspender/commit/b7157c38db080fceca29e20378022c007ddaf765), in "js_session_save_point" used russian word restore by analogy with Windows automatic restore points but original is ok too, waiting on ur opinions.

### CODE REVIEW

File: src/_locales/ru/messages.json [hunk 1]
```diff
@@ -134,9 +134,9 @@
     "html_popup_suspend_selected_tabs": { "message": "Приостановить выбранные вкладки" },
     "html_popup_unsuspend_selected_tabs": { "message": "Возобновить выбранные вкладки" },
     "html_popup_settings": { "message": "Настройки" },
-    "html_restoring_title": { "message": "Restoring window" },
-    "html_restoring_heading": { "message": "Restoring tabs" },
-    "html_restoring_detail": { "message": "Please wait while I restore this window..." },
+    "html_restoring_title": { "message": "Восстановление окна" },
+    "html_restoring_heading": { "message": "Восстановление вкладок" },
+    "html_restoring_detail": { "message": "Пожалуйста, подождите пока я восстановлю это окно..." },
```
Comment (AlexWayfer) [inline src/_locales/ru/messages.json]:
```suggestion
    "html_restoring_detail": { "message": "Пожалуйста, подождите, пока я восстановлю это окно..." },
```

(complicated sentence: "[you] wait" + "I restore", have to be separated by comma)

**Comment by AlexWayfer** [comment]
@deanoemcke, maybe `"js_session_save_point": { "message": "Automatic save point for v" }` should be `"Automatic save point for version "` for more obvious.

**Comment by Jollfye** [comment]
> @deanoemcke, maybe `"js_session_save_point": { "message": "Automatic save point for v" }` should be `"Automatic save point for version "` for more obvious.

@AlexWayfer hmm to me current "...v" seems ok, it' shorter, minimalistic, I like it that way, but I get ur point when even Chrome on the page chrome://settings/help uses full word "version"

**Comment by AlexWayfer** [comment]
> @AlexWayfer hmm to me current "...v" seems ok, it' shorter, minimalistic, I like it that way

Maybe then in Russian we should use `"Автоматическая точка восстановления для вер."` :man_shrugging: 

> but I get ur point when even Chrome on the page chrome://settings/help uses full word "version"

Not bad.

Also, I'm interested now, how it looks in Chrome in Russian (I'm using English as system language).

**Comment by Jollfye** [comment]
@AlexWayfer me too + I'm also interested, maybe I'll give it a try to see in action 😅 And I like ur suggestion to use contraction for russian version but I'm not sure, let's wait for @MonsterNya, what he may say here

**Comment by AlexWayfer** [comment]
> And I like ur suggestion to use contraction for russian version but I'm not sure, let's wait for @MonsterNya, what he may say here

Just a note for English-speakers: `для вер.` is equal to `for ver.`

`в` (like `v`) is not common in Russian for versions.

So, I suggested the complete form of a word: `version`/`версии`

**Comment by deanoemcke** [comment]
@AlexWayfer I'm ok with using the complete word `version`

**Comment by MonsterNya** [comment]
I came after the party ended, but I am pleased with the way she went ^_^

**Comment by Jollfye** [comment]
Alright then, let's leave "js_session_save_point": { "message": "Автоматическая точка восстановления для версии " } as is in russian, and @deanoemcke may change English if he's ok with it.

**Comment by AlexWayfer** [comment]
> Alright then, let's leave "js_session_save_point": { "message": "Автоматическая точка восстановления для версии " } as is in russian, and @deanoemcke may change English if he's ok with it. Aand I guess that's it with this PR here.

I agree.

**Pull request merged**